src/task2.js

```diff
@@ -0,0 +1,45 @@
+const csv = require('csvtojson');
+const fs = require('fs');
+
+const writeText = fs.createWriteStream('node_mentoring_t1_2_input_example.txt');
+const results = [];
+
+var s = fs.createReadStream('node_mentoring_t1_2_input_example.csv')
+        .pipe(csv())
+        .on('data', (data) =>  writeText.write(data))
+        .on('error', function(err){
+            console.log('Error while reading file.', err);
+        })
+        .on('end', function(){
+            console.log('After File Content Checked.');
+        });
+
+
```

Comment:
Also, please remove any non-related to the final version of your task code.
This will improve an overall quality of your project :)

Reply:
Updated as per the comment